OWNERS

```diff
@@ -0,0 +1,12 @@
+reviewers:
```

Comment:
that's a huge list. Can we reduce it to those who worked on this, especially the approvers? :)